scout/artifactory.md

```diff
@@ -160,6 +160,27 @@ $ docker run \
   docker/artifactory-agent:v1
 ```
 
+#### Analyzing pre-existing data
+
+By default the agent will detect and analyze images as they're created and
+updated. If you would like to use the agent to analyze pre-existing images, you
+can use backfill mode. Use the `--backfill-from=TIME` command line argument,
```

Comment:
```suggestion
can use backfill mode. Use the `--backfill-from=TIME` command line option,
```

@jonnystoten We haven't mentioned this anywhere, but standardising the language we use around command line "things" came up in a conversation elsewhere and hasn't been recorded yet ;)